### PR TITLE
Convert terraform registry module references to local filesystem references

### PR DESCRIPTION
**Pull request by mkvalor-AL**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -15,16 +15,16 @@ const (
 	VERSION = "v1.2.0"
 
 	// TFQueueModuleVersion represents the version of the hedwig-queue module
-	TFQueueModuleVersion = "1.2.0"
+	//TFQueueModuleVersion = "1.2.0"
```
Comment (automatic-glen-jarvis) [inline main.go]: @mkvalor-AL Did you mean to keep these versions commented out for this PR?

Is this related to the Terraform hosted PRs?

Do we want to make a comment instead here so that we know it's local to the repo instead? (assuming the above is true)

Comment (mkvalor-AL) [reply main.go]: Good catch.  I had left the commented lines in to make sure I could remember the var names to chase down in the rest of the code.  Now that's done, I'll delete these.

Comment (automatic-glen-jarvis) [reply main.go]: 👍
